test/end-to-end/flags.spec.js

```diff
@@ -32,4 +34,28 @@ describe('contentful-migration CLI flags', function () {
       .expect(assert.confirmation.noConfirmationMessage())
       .end(done);
   });
+
+  describe('file path resolution', function () {
+    // Use the Travis build dir for integration tests, fallback to cwd for local tests
+    const buildDir = process.env.TRAVIS_BUILD_DIR || process.cwd();
+
+    it('does not treat absolute script path as relative', function (done) {
+      const scriptPath = path.join(buildDir, 'test/end-to-end/migration-scripts/non-existing-migration.js');
+
+      cli()
+        .run(`--space-id ${SOURCE_TEST_SPACE} ${scriptPath}`)
+        .expect(assert.help.nonExistingMigrationScript(scriptPath))
+        .end(done);
+    });
+
+    it('resolves relative script path to cwd', function (done) {
+      const scriptPath = 'test/end-to-end/migration-scripts/non-existing-migration.js';
+      const resolvedPath = path.join(buildDir, scriptPath);
```

Comment:
🙈  how is this path different to the one in line 43?

Reply:
I also had the same question 😄

Reply:
First test gets an "already resolved" (aka: absolute) path and expects it to not change.
This test uses a relative path and expects the CLI to resolve it to an absolute path.

Reply:
Ah, I see. We are testing the error response 👍